### DISCUSSION TITLE
Meeting of Remix CC (2023-10-15)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#92] [Meeting of Remix CC]
### 议题
暂无

**Comment by drafish** [comment]
### DJQ
1. Q3总结+Q4规划
2. GCC grants application for Q4
3. 与星火链签了合同

**Comment by yiwen4** [comment]
### yiwen
工作进度
1. 整理RCC运营相关内容，细化运营计划，和普朗克运营小组开会、
  梳理出RCC运营计划重点——品牌形象塑造、内容整理（自我介绍、愿景、logo设计）
2. 写一篇教学视频制作指南
3. review9月翻译

提议议题
1.Q4工作规划报名讨论，是否需要开放为bounty任务
2.确定是否设计品牌logo
3.各平台账号手机号注册用DJQ的？
4.remix的受众群体
  建议的插件，https://cryptozombies.io/
5.异步沟通，具体时间节点需提前提出
